extensions/replication/queueProcessor/ReplicationTaskScheduler.js

```diff
@@ -7,16 +7,16 @@ const async = require('async');
 class ReplicationTaskScheduler {
```

Comment:
Can you also update the description above 

 /**
  * Schedule replication tasks for replicated buckets, by ensuring that
  * all updates to the same ~~version of an~~ object are serialized
 */

Reply:
done